src/bp/core/api.ts

```diff
@@ -17,6 +17,7 @@ import HTTPServer from './server'
 import { GhostService } from './services'
 import { BotService } from './services/bot-service'
 import { CMSService } from './services/cms'
+import { ConverseService } from './services/converse'
```

Comment:
Those changes are no longer related to channel messenger. I'm not sure we still want to expose a converse service in the botpress sdk (especially because it's available through http)

Reply:
I think its nice to expose converse to use if need be. As the doc states, proactive won't work with converse, so users that use it should know.